src/measure/OSMeasureInfoGetter.cpp

```diff
@@ -161,7 +161,27 @@ namespace measure {
       boost::optional<std::string> minValue;
       boost::optional<std::string> maxValue;
       if (argument.hasDomain()) {
-        // TODO
+        if (argument.type() == OSArgumentType::Integer) {
+          auto domain = argument.domainAsInteger();
+          auto& low = domain.front();
+          auto& high = domain.back();
+          if (low > std::numeric_limits<int>::lowest()) {
+            minValue = std::to_string(low);
+          }
+          if (high < std::numeric_limits<int>::max()) {
+            maxValue = std::to_string(high);
+          }
+        } else if (argument.type() == OSArgumentType::Double) {
+          auto domain = argument.domainAsDouble();
+          auto& low = domain.front();
+          auto& high = domain.back();
+          if (low > std::numeric_limits<double>::lowest()) {
+            minValue = std::to_string(low);
+          }
+          if (high < std::numeric_limits<double>::max()) {
+            maxValue = std::to_string(high);
+          }
+        }
```

Comment:
I noticed this TODO so I also addressed it. It will get passes to BCLMeasureArgument => the `measure.xml` now will also include the min/max values


![image](https://user-images.githubusercontent.com/5479063/176426252-bf68e9f3-00b5-4655-8dc1-8fde9e54b4b3.png)